src/test/AuxData.test.cpp

```diff
@@ -481,7 +481,7 @@ TEST(Unit_AuxData, TupleofVariants) {
   using VarT = std::variant<Addr, char>;
   using STH = gtirb::SerializationTestHarness;
   AuxDataImpl<TupleOfCharOrAddr> Original =
-      std::tuple<VarT, VarT, VarT>{Addr(0xc0ffee), 'z', 0x1};
+      std::tuple<VarT, VarT, VarT>{Addr(0xc0ffee), 'z', (char)0x1};
```

Comment:
I'm not sure what the point is for having the third item in the tuple, anyways. It's value is not checked, and it seems like it would just be duplicating whatever is tested by the second item in the tuple. So perhaps a better change would be to just make `TupleOfCharOrAddr` be a 2-tuple instead of a 3-tuple. It's not used anywhere else.

If we're going to keep it, though, I think the preference is to use `static_cast<char>(0x1)` rather than the C-style cast.